cedar-policy-core/src/entities/json/entities.rs

```diff
@@ -72,20 +83,28 @@ enum EntitySchemaInfo<E: EntityTypeDescription> {
     NonAction(E),
 }
 
-impl<'e, S: Schema> EntityJsonParser<'e, S> {
+impl<'e, 's, S: Schema> EntityJsonParser<'e, 's, S> {
     /// Create a new `EntityJsonParser`.
     ///
-    /// If a `schema` is present, this will inform the parsing: for instance, it
-    /// will allow `__entity` and `__extn` escapes to be implicit.
-    /// It will also ensure that the produced `Entities` fully conforms to the
-    /// `schema` -- for instance, it will error if attributes have the wrong
-    /// types (e.g., string instead of integer), or if required attributes are
-    /// missing or superfluous attributes are provided.
+    /// `schema` represents a source of `Action` entities, which will be added
+    /// to the entities parsed from JSON.
+    /// (If any `Action` entities are present in the JSON, and a `schema` is
```

Comment:
Same comment as above

Reply:
Updated the wording with a somewhat different change, hopefully things are clear now

Reply:
+1 to the original comment. Isn't this text the same as the doc string for the `schema` field?

Reply:
What does your +1 mean -- do you not think that the current wording (after my update) is clear?

Andrew's suggestion includes "the same action entities must be listed" which is not quite true, because we (both previously and now) allow passing a subset of the schema's actions in the entities.  Whatever actions are passed must exactly match their schema definitions.  I tried to make this clear with my update, while still keeping things relatively concise (this doc comment is already long).

Yes, this text is the same as the doc string for the `schema` field.  But, the `schema` field is private, so this doc serves (in principle) to inform public (aka, outside-of-Core) users of what the `schema` argument to `new()` does.

Reply:
I also worry about that; do you have a suggested alternative?

Reply:
(Andrew wasn't referring to the fact that it's duplicate text IIUC; he was saying that his above comment also applies here)

Reply:
Ah, misinterpreted Andrew's comment then 😆 Since the `new` function is the public one, I would suggest leaving the doc string there and then update the `schema` field doc string to say "see `EntityJsonParser::new()`

Reply:
Done